[2025-05-21T00:12:52.862+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-21T00:12:52.906+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: merge_sources_pipeline.preprocessing manual__2025-05-21T00:12:38.907965+00:00 [queued]>
[2025-05-21T00:12:52.926+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: merge_sources_pipeline.preprocessing manual__2025-05-21T00:12:38.907965+00:00 [queued]>
[2025-05-21T00:12:52.927+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-21T00:12:52.950+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): preprocessing> on 2025-05-21 00:12:38.907965+00:00
[2025-05-21T00:12:52.970+0000] {standard_task_runner.py:72} INFO - Started process 428 to run task
[2025-05-21T00:12:52.979+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'merge_sources_pipeline', 'preprocessing', 'manual__2025-05-21T00:12:38.907965+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/inference_dag.py', '--cfg-path', '/tmp/tmpsp1sdnp1']
[2025-05-21T00:12:52.985+0000] {standard_task_runner.py:105} INFO - Job 68: Subtask preprocessing
[2025-05-21T00:12:53.083+0000] {task_command.py:467} INFO - Running <TaskInstance: merge_sources_pipeline.preprocessing manual__2025-05-21T00:12:38.907965+00:00 [running]> on host 7f42758d044f
[2025-05-21T00:12:53.250+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='merge_sources_pipeline' AIRFLOW_CTX_TASK_ID='preprocessing' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T00:12:38.907965+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-21T00:12:38.907965+00:00'
[2025-05-21T00:12:53.253+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-21T00:12:53.281+0000] {warnings.py:109} WARNING - /opt/***/scripts/preprocess.py:41: UserWarning: Could not infer format, so each element will be parsed individually, falling back to `dateutil`. To ensure parsing is consistent and as-expected, please specify a format.
  df["Take-off Time (UTC)"] = pd.to_datetime(df["Take-off Time (UTC)"], errors='coerce')

[2025-05-21T00:12:53.304+0000] {python.py:240} INFO - Done. Returned value was: []
[2025-05-21T00:12:53.350+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-21T00:12:53.351+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=merge_sources_pipeline, task_id=preprocessing, run_id=manual__2025-05-21T00:12:38.907965+00:00, execution_date=20250521T001238, start_date=20250521T001252, end_date=20250521T001253
[2025-05-21T00:12:53.435+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-21T00:12:53.485+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-21T00:12:53.488+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
